Jenkinsfile

```diff
@@ -0,0 +1,70 @@
+pipeline {
```

Comment:
We have moved to travis.yaml, so Jenkinsfile should be removed ?
@ClementGautier  @samlesu

Reply:
Yup, you should probably rebase (it's not present on dev anymore)